### PR TITLE
zh-trans: modify /docs/reference/glossary/taint.md

### PR DESCRIPTION
**Pull request by lpf7551321**
zh-trans: modify /docs/reference/glossary/taint.md
Update: /docs/reference/glossary/taint.md
Co-Authored-By: lpf7551321

### CODE REVIEW

File: content/zh/docs/reference/glossary/taint.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ A core object consisting of three required properties: key, value, and effect. Taints prevent the scheduling of pods on nodes or node groups.
+-->
+
+<!--more-->
+
+<!--
+Taints and {{< glossary_tooltip text="tolerations" term_id="toleration" >}} work together to ensure that pods are not scheduled onto inappropriate nodes. One or more taints are applied to a {{< glossary_tooltip text="node" term_id="node" >}}. A node should only schedule a pod with the matching tolerations for the configured taints.
+-->
+
+污点和 {{< glossary_tooltip text="容忍" term_id="toleration" >}} 一起工作，以确保不会将 Pod 调度到不适合的节点上。一个或多个污点应用于 {{< glossary_tooltip text="节点" term_id="node" >}}。节点应该仅能调度那些带着能与污点相匹配的容忍（tolerations）的 pod。
```
Comment (xichengliudui) [inline content/zh/docs/reference/glossary/taint.md]: 容忍 ->  容忍度
ref：https://github.com/k8smeetup/k8s-official-translation/blob/master/trans-glossary.md

Comment (lpf7551321) [reply content/zh/docs/reference/glossary/taint.md]: ok

**Comment by xichengliudui** [comment]
/lgtm
/hold cancel
@lpf7551321  thanks!

**Pull request merged**